3-UsingAPIs/Week2/assignment/ex2-pokemonApp/index.js

```diff
@@ -24,20 +24,16 @@ Try and avoid using global variables. As much as possible, try and use function
 parameters and return values to pass data back and forth.
 ------------------------------------------------------------------------------*/
 async function fetchData(url) {
-  try {
-    const response = await fetch(url);
-    if (!response.ok) {
-      throw new Error('HTTP or network errors');
-    }
-    const pokemon = await response.json();
-    return pokemon;
-  } catch (error) {
-    throw new Error('HTTP or network errors', error);
+  const response = await fetch(url);
+  if (!response.ok) {
+    throw new Error('HTTP or network errors');
   }
+  return await response.json();
 }
 
 async function fetchAndPopulatePokemons() {
   const select = document.querySelector('.select-pokemon');
+  select.textContent = '';
```

Comment:
Although this seems to work you don't want to just clear the text content but the inner HTML content.
```suggestion
  select.innerHTML = '';
```